owncloudTestUtil/src/main/java/com/owncloud/android/testutil/OCFile.kt

```diff
@@ -83,3 +83,4 @@ val OC_FILES_WITH_SYNC_INFO = listOf(OC_FILE_WITH_SYNC_INFO, OC_FILE_WITH_SYNC_I
 val OC_AVAILABLE_OFFLINE_FILES = listOf(OC_AVAILABLE_OFFLINE_FILE, OC_AVAILABLE_OFFLINE_FILE, OC_AVAILABLE_OFFLINE_FILE)
 val OC_FILES_EMPTY = emptyList<OCFile>()
 val OC_FILES_WITH_SYNC_INFO_EMPTY = emptyList<OCFileWithSyncInfo>()
+val OC_FILE_SPACE_WEB_DAV_URL = "url"
```

Comment:
I think you can use here the `webUrl` property of one of the space objects we already have in the `OCSpace.kt` file in the owncloudTestUtil module, so that it is more realistic and we don't need to create new variables. For example, you can use `OC_SPACE_PROJECT_WITH_IMAGE.webUrl` 👍